rust/arrow/src/compute/kernels/arithmetic.rs

```diff
@@ -274,6 +345,19 @@ mod tests {
         assert_eq!(9, c.value(4));
     }
 
+    #[test]
+    fn test_primitive_array_divide_with_nulls() {
```

Comment:
I would suggest to use template to covert all numeric types in test.

Reply:
@liurenjie1024 I'm not sure what you mean, could you explain?

Reply:
I mean maybe we should cover all numerics types, not just int32?

Reply:
Ok, yep good point.  Follow up JIRA is [here](https://issues.apache.org/jira/browse/ARROW-6409?filter=-2)